src/main/java/entity/Player.java

```diff
@@ -434,11 +432,7 @@ public final void setLives(final int lives) {
 	public final void setPlayerTwo(final boolean playerTwo) {
 		this.playerTwo = playerTwo;
 		if (getThisGame().getGamestate().isCoop()) {
-			if (playerTwo) {
-				this.playerString = "Player 2";
-			} else {
-				this.playerString = "Player 1";
-			}
+			this.playerString = (playerTwo) ? "Player 2" : "Player 1";
```

Comment:
you haven't removed if statements here, an inline conditional is still an if else statement. additionally it is les readable like this. unless the TA feels like this is alright please revert this back to the way it was.